multi-agent/lib/mas/elements/nodes/llm_merger/config.py

```
@@ -42,5 +46,5 @@ class MergerLLMNodeConfig(NodeBaseConfig):
         Output:
         - Start with the merged answer using inline agent tags and source mentions.
         - End with the improvement section if needed.
-        - Use a professional, neutral tone that’s easy to follow.""",
+        -         Use a professional, neutral tone that’s easy to follow.""",
```

Ruff (0.15.21)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)